doc/nrf/releases/release-notes-latest.rst

```diff
@@ -91,6 +91,36 @@ The following list summarizes the most important changes inherited from upstream
 
   * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
 
+* Storage and filesystems:
+
+  * A possible NULL pointer dereference when using any of the ``fs_`` functions has been fixed and will not return an error code instead.
+  * A garbage-collection issue has been fixed in the NVS subsystem.
+
+* Devicetree:
+
+  * All nRF-specific aliases with a dash (``-``) have been removed since they
+    are no longer needed (eg. ``i2c-0``, ``spi-0``, etc).
```

Comment:
```suggestion
  * Removed all nRF-specific aliases with a dash (``-``, for example, ``i2c-0`` or ``spi-0``), because they are no longer needed.
```

Reply:
It's not the presence or absence of a dash which tells you if the alias was removed. Aliases that refer to a specific hardware peripheral instance were removed. The presence of a dash in the alias names is not directly relevant. This affects devicetree nodes for all peripheral instances, including SPI, I2C, UART, ADC, GPIO, GPIOTE, WDT, QDEC, RTC, TIMER, PWM, cryptocell, USB, FLASH, PDM, and EGU peripherals.

The aliases were defined because the legacy devicetree macros prevented users from referring to nodes by "node label", but allowed reference by alias.

The new devicetree.h API allows referring to nodes by node label, so the aliases were removed to avoid redundancy and confusion going forward.

For example, users should use `DT_NODELABEL(i2c0)` instead of `DT_ALIAS(i2c_0)`.